app/models/Product.coffee

```diff
@@ -4,3 +4,15 @@ module.exports = class ProductModel extends CocoModel
   @className: 'Product'
   @schema: require 'schemas/models/product.schema'
   urlRoot: '/db/products'
+
+  priceStringNoSymbol: -> (@get('amount') / 100).toFixed(2)
+
+  purchase: (token, options={}) ->
+    options.url = _.result(@, 'url') + '/purchase'
+    options.method = 'POST'
+    options.data ?= {}
+    options.data.token = token.id
+    options.data.timestamp = new Date().getTime()
+    options.data = JSON.stringify(options.data)
+    options.contentType = 'application/json'
+    return $.ajax(options)
```

Comment:
Mm, a proper solution for migrating toward the fetch API is turning out to be out of scope. Gonna leave this as is and trust the tests will make sure network errors and such are handled properly.